app/views/dashboards/show.html.slim

```diff
@@ -0,0 +1,18 @@
+.p20
+  h2
+    | Dashboard Page
+  .mt30.p20
+    p.strong
+      = image_tag @customer['photo']
+    p.strong
+      |Name:&nbsp;&nbsp;
```

Comment:
Using `&nbsp;` is a bad practice and not modern. I suggest to use CSS rules for make lines non-breaking and add required paddings/margins between words if you need.